### PR TITLE
Fixed #1194 : Replaced periods with single-character ellipsis in messages for consistency.

### LINKED ISSUE #1194: Inconsistent use of periods vs ellipses in messages
# Issue summary

Some messages in the CLI use `...` while others use `…` and this should be consistent throughout.
@gfscott Which should it be?

### PR DESCRIPTION
**Pull request by sharvy**
### WHY are these changes introduced?

Both periods and single character ellipsis were used in messages, which resulted in inconsistent pattern.

Fixes #1194 

### WHAT is this pull request doing?

This pull request is replacing periods with single-character ellipsis, in 5 different messages files.

### Update checklist

- [x] I've added a CHANGELOG entry for this PR (if the change is public-facing)
- [x] I've considered possible cross-platform impacts (Mac, Linux, Windows).

### CODE REVIEW

**Comment by sharvy** [comment]
I would appreciate your feedback 🙂

cc: @andyw8  @gfscott  @tatheerf02

File: lib/shopify-cli/messages/messages.rb [hunk 1]
```diff
@@ -334,7 +334,7 @@ module Messages
           start: "{{v}} ngrok tunnel running at {{underline:%s}}",
           start_with_account: "{{v}} ngrok tunnel running at {{underline:%s}}, with account %s",
           stopped: "{{green:x}} ngrok tunnel stopped",
-          timed_out: "{{x}} ngrok tunnel has timed out, restarting ...",
+          timed_out: "{{x}} ngrok tunnel has timed out, restarting …",
```
Comment (andyw8) [inline lib/shopify-cli/messages/messages.rb:337]: Can we drop the space before the `…`? (I realise it was already there, but seems more noticeable now).

Comment (sharvy) [reply lib/shopify-cli/messages/messages.rb]: @andyw8 Done, and I checked, there isn't any other similar issue.

**Review by gfscott — APPROVED** [review]
This is great, thanks! 👍🏻

**Comment by mkevinosullivan** [comment]
👋🏻 Thank you @sharvy for the contribution - I've replicated, augmented and merged it in via #1343, as the codebase has changed quite a bit for `v2` of the CLI.

**Pull request closed without merging**